tools/SourceKit/lib/SwiftLang/CodeCompletionOrganizer.cpp

```diff
@@ -661,13 +657,13 @@ static ResultBucket getResultBucket(Item &item, bool hasRequiredTypes,
   if (completion->isNotRecommended() && !skipMetaGroups)
     return ResultBucket::NotRecommended;
 
-  if (completion->getSemanticContext() ==
-          SemanticContextKind::ExpressionSpecific &&
-      !skipMetaGroups)
-    return ResultBucket::ExpressionSpecific;
-
-  if (completion->isArgumentLabels() && !skipMetaGroups)
-    return ResultBucket::ExpressionSpecific;
+  if (!skipMetaGroups) {
+    auto flair = completion->getFlair();
+    if (flair.contains(CodeCompletionFlairBit::ExpressionSpecific) ||
```

Comment:
Should this just check for non-zero?

Reply:
In https://github.com/apple/swift/pull/37725 and in future I'm going to introduce other values that should not be in `ExpressionSpecific` bucket.